.travis.yml

```diff
@@ -6,8 +6,19 @@ notifications: # set notification options
 sudo: false
 
 node_js:
-  - "4.1"
+  - "5.4.1"
```

Comment:
I was having trouble getting node-gyp builds to work, so I was trying to make Travis as cloase to my local machine as possible. It can be changed back to 5 now that I figured out the actual problem